### DISCUSSION TITLE
App crashes after latest update

### DISCUSSION THREAD

**Discussion by NadCat1** [bttv-android/bttv#414] [General]
Not sure if this is the right place to post this but after the latest update the app keeps crashing after some time (not sure how long, it could be anything between 10-30 minutes). Downgrading to 21.0 fixed it

Also another question: is there a special setting for notifications I'm not finding? I'm not getting any notifications about channels going live (Smart notifications are disabled, it worked with the Twitch app)

**Comment by FoseFx** [comment]
This sounds worrisome. I need a bit more information because this does not happen to me.

We are talking about v0.22.2-beta correct?
Does the same thing happen with the unmodded Twitch App v12.4.1?
Can you describe what you do until this happens? Like step-by-step instructions for me to reproduce.
Is it a crash ("bttv-android was stopped") or a ANR ("Application is not responding")?
And lastly it would be great if you can share some device information. Which device do you use? What android version is it running?

**Comment by NadCat1** [comment]
Yes, v0.22.2-beta is the one that crashes
I didn't notice any crashes with the Twitch App but I don't think I've used v12.4.1, I switched to BTTV mobile before that update
I open the app and open a stream. After some time the app gets really slow and then crashes
I'll have to update again and see what kind of crash it is
I'm using a Fairphone 4 with the LOS beta based on Android 11

Do you have any info about the notifications? After reinstalling v0.21 I instantly got a "New update" notification but I'm still not getting any live notifications

**Reply by FoseFx** [reply]
My best guess is memory is not cleared up correctly by the system. Maybe the update changed something related to rendering which now causes this for you, but there is no easy way to verify this for me.

AFAIK Lineage OS (which I assume LOS stands for) uses microG instead of the official Google Play Services which include Firebase Cloud Messaging which is used for sending/receiving notifications, maybe there is an issue with that.

<details>
<summary>It's working fine on my system. Screenshot:</summary>
<img src="https://user-images.githubusercontent.com/22842759/156143794-f8dfd864-3462-4c58-a936-36ba6bfdb11d.jpg">
</details>

**Reply by NadCat1** [reply]
I've let it run for 2+ hours now and it still hasn't crashed. Maybe the first installation had an error.

Yeah, it is LineageOS. It's a build with GApps preinstalled (the Twitch App notifications worked)
